test/EndToEndTests/Tests/Client/Build.Desktop/ClientWithoutTypeResolverTests/JsonWithoutTypeResolverTests.cs

```diff
@@ -86,26 +86,26 @@ public void NestedComplexPropertyQuery()
             var queryResults4 = contextWrapper.Execute<ICollection<string>>(new Uri(this.ServiceUri.OriginalString + "/Customer(-10)/Microsoft.Test.OData.Services.AstoriaDefaultService.Customer/PrimaryContactInfo/ContactAlias/AlternativeNames")).ToArray();
         }
 
-        [TestMethod]
+        [Fact]
         public void CollectionOfComplexPropertyQuery()
         {
             var contextWrapper = this.CreateContext();
             var queryResults = contextWrapper.Execute<Phone>(new Uri(this.ServiceUri.OriginalString + "/Customer(-10)/PrimaryContactInfo/MobilePhoneBag")).ToArray();
         }
 
-        [TestMethod]
+        [Fact]
         public void CollectionOfPrimitivePropertyQuery()
         {
             var contextWrapper = this.CreateContext();
             var queryResults = contextWrapper.Execute<ICollection<decimal>>(new Uri(this.ServiceUri.OriginalString + "/MappedEntityType(-10)/BagOfDecimals")).ToArray();
         }
 
-        [TestMethod]
+        [Fact]
         public void ServiceOperationFeedQuery()
         {
             var contextWrapper = this.CreateContext();
             var queryResult = contextWrapper.Execute<Customer>(new Uri(this.ServiceUri.OriginalString + "/GetSpecificCustomer?Name='enumeratetrademarkexecutionbrfalsenesteddupoverflowspacebarseekietfbeforeobservedstart'"), "GET", true).ToArray();
-            Assert.AreEqual(1, queryResult.Count(), "Expected a single Customer return");
+            Assert.Equal(1, queryResult.Count());

```

Comment:
```suggestion
            Assert.Single(queryResult);

```

Reply:
xunit recommends the use of Single for a collection with a count of 1.

Reply:
Isn't that what this is?